.github/workflows/validate.yml

```diff
@@ -0,0 +1,74 @@
+name: Validate
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  typecheck:
+    name: ʦ TypeScript (${{ matrix.project }})
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        project: [blob, fetch, file, form-data, stream]
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v3
+
+      - name: ⎔ Setup node
+        uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version: 16
```

Comment:
Should this be 18 so it's current LTS?

Reply:
I kept it at 16, just the way it was

@jacob-ebey's PR can upgrade to 18 I guess?